app/components/document/index/filter_options_component.rb

```diff
@@ -9,6 +9,20 @@ def initialize(filters:, errors: nil)
 
 private
 
+  def heading
+    "Search by keyword"
+  end
+
+  def data_attributes
```

Comment:
The functionality here is a lot like the `FormHelper.ga4_data_attributes` function. I realise this has slightly different values but this also has a different level of nesting too. Just wondering if it's worth making it more uniform with the others?